packages/pwa-kit-react-sdk/src/ssr/server/opentelemetry-server.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2025, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {NodeTracerProvider} from '@opentelemetry/sdk-trace-node'
+import {SimpleSpanProcessor} from '@opentelemetry/sdk-trace-base'
+import {B3Propagator} from '@opentelemetry/propagator-b3'
+import {Resource} from '@opentelemetry/resources'
+import {propagation} from '@opentelemetry/api'
+import logger from '../../utils/logger-instance'
+
+const DEFAULT_SERVICE_NAME = 'pwa-kit-react-sdk'
+
+let provider = null
+
+/**
+ * Initialize OpenTelemetry tracing for server-side rendering
+ * @param {Object} options
+ * @param {string} [options.serviceName]
+ * @param {string} [options.serviceVersion]
+ * @param {boolean} [options.enabled]
+ * @returns {NodeTracerProvider|null}
+ */
+export const initializeServerTracing = (options = {}) => {
+    const {
+        serviceName = process.env.OTEL_SERVICE_NAME || DEFAULT_SERVICE_NAME,
+        serviceVersion = process.env.npm_package_version,
```

Comment:
@larnelleankunda Thanks for adding the telemetry options! It was my bad for not clarifying the code I shared was just an example of possible configuration options. Since `process.env.npm_package_version` will always be undefined, let’s remove it for now. We can explore a better way to include it later if needed.

```suggestion
```